tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -560,3 +560,68 @@
 - name: Adopted Neutron and OVN agents post-checks
   ansible.builtin.include_tasks:
     file: neutron_verify.yaml
+
+- name: Remove leftover OOO services
+  block:
+    - name: Create OpenStackDataPlaneService/tripleo-cleanup
+      no_log: "{{ use_no_log }}"
+      ansible.builtin.shell: |
+        {{ shell_header }}
+        {{ oc_header }}
+        oc apply -f - <<EOF
+        apiVersion: dataplane.openstack.org/v1beta1
+        kind: OpenStackDataPlaneService
+        metadata:
+          name: tripleo-cleanup
+        spec:
+          playbook: osp.edpm.tripleo_cleanup
```

Comment:
I wonder if we should have it in the dataplane-operator repo

Reply:
We have priory art defining adoption specific service on the fly in the data-plane-adoption repo. The pre-adoption-validation service is defined in the same way.